README.md

```diff
@@ -18,7 +18,7 @@ The Open Source Next.js SaaS boilerplate for Enterprise SaaS app development.
 
 ## Additional Resources
 
-Video - [Live Discord Community Call Demo: Build a SaaS Application with BoxyHQ's SaaS Starter Kit](https://www.youtube.com/watch?v=oxbEHmhTYQo) <br> Blog - [Enterprise-ready Saas Starter Kit](https://boxyhq.com/blog/enterprise-ready-saas-starter-kit)
+Video - [Live Discord Community Call Demo: Build a SaaS Application with BoxyHQ's SaaS Starter Kit](https://studio.youtube.com/video/oF8QIwQIhyo/edit?c=UCRx3EWQL1l07MTgtDHx5IMA) <br> Blog - [Enterprise-ready Saas Starter Kit](https://boxyhq.com/blog/enterprise-ready-saas-starter-kit)
```

Comment:
@NathanTarbert Looks like the YT video link is wrong - pointing it to studio URL.